modelica_builder/edit.py

```diff
@@ -92,8 +92,13 @@ def apply_edits(cls, edits, document):
         # entire document for every edit
         # O(m*n), m = bytes in document, n = number of edits
         for edit in ordered_edits:
-            # remove edit span
-            if edit.start != edit.stop:
+            # catch the case where the name is to replaced which is only one character. The `get_span` method
+            # will set the edit.stop to None if the length is only one.
+            if edit.stop is None:
```

Comment:
This section should really be handled in the `get_span` method.

Reply:
I just added a fix for this in get_span - it now returns a range which is more pythonic, where the stop == index of end of node text + 1.